test/hotspot/jtreg/compiler/c2/aarch64/TestVectorShiftShorts.java

```diff
@@ -27,7 +27,8 @@
  * @bug 8255287
  * @summary aarch64: fix SVE patterns for vector shift count
  *
- * @requires os.arch == "aarch64" & vm.compiler2.enabled
+ * @requires vm.compiler2.enabled
+ * @requires os.arch == "aarch64" | os.arch == "riscv64"
```

Comment:
Seems a bit weird to enable tests under `test/hotspot/jtreg/compiler/c2/aarch64/` for other cpus?

Reply:
Yeh, it's bit, removed ` | os.arch == "riscv64"`.

I can also modify the summary and enable the test for riscv if needed.